instrumentation/jboss-logmanager/jboss-logmanager-mdc-1.1/javaagent/src/test/groovy/JbossLogmanagerMdcTest.groovy

```diff
@@ -80,6 +80,10 @@ class JbossLogmanagerMdcTest extends AgentInstrumentationSpecification {
     logRecords.get(2).getMdc("trace_id") == span1.spanContext.traceId
     logRecords.get(2).getMdc("span_id") == span1.spanContext.spanId
     logRecords.get(2).getMdc("trace_flags") == "01"
+    def mdcCopy2 = logRecords.get(2).getMdcCopy()
+    mdcCopy2.get("trace_id") != null
```

Comment:
would it be possible to validate against expected value?

Reply:
Updated it to test against the expected value.